# Awesome papers and projects about CNV and SV using NGS data :books:

Relevant studies with Structual Variants and Copy Number Variants in NGS (Genome, Exome and Amplicon Sequencing) pipelines.
---

## Background CNV and SV
* [2008 Integrated detection and population-genetic analysis of SNPs and copy number variation](https://www.nature.com/articles/ng.238)
* [2010 Origins and functional impact of copy number variation in the human genome](https://www.ncbi.nlm.nih.gov/pubmed/19812545)
* [2011 Genome structural variation discovery and genotyping](http://www.nature.com/nrg/journal/v12/n5/full/nrg2958.html?foxtrotcallback=true)
* [2011 Modeling read counts for CNV detection in exome sequencing data](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3517018/)
* [2011 American College of Medical Genetics standards and guidelines for interpretation and reporting of postnatal constitutional copy number variants](https://www.nature.com/articles/gim92011110)
* [2012 Read count approach for DNA copy number variants detection](https://academic.oup.com/bioinformatics/article/28/4/470/213239)
* [2013 Computational tools for copy number variation (CNV) detection using next-generation sequencing data: features and perspectives](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S11-S1)
* [2014 An Evaluation of Copy Number Variation Detection Tools from Whole-Exome Sequencing Data](http://onlinelibrary.wiley.com/doi/10.1002/humu.22537/abstract)
* [2014 Refining analyses of copy number variation identifies specific genes associated with developmental delay](http://www.nature.com/ng/journal/v46/n10/abs/ng.3092.html)
* [2015 Use of high-throughput targeted exome-sequencing to screen for copy number variation in hypertrophic cardiomyopathy](https://www.sciencedirect.com/science/article/pii/S1769721215300252?via%3Dihub) 
> An example using ExomeDepth
* [2015 Allele-specific copy-number discovery from whole-genome and whole-exome sequencing](https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkv319)
* [2015 Copy number variant analysis from exome data in 349 patients with epileptic encephalopathy](http://onlinelibrary.wiley.com/doi/10.1002/ana.24457/abstract)
* [2015 Evaluation of somatic copy number estimation tools for whole-exome sequencing data](https://academic.oup.com/bib/article-abstract/17/2/185/1744035/Evaluation-of-somatic-copy-number-estimation-tools?redirectedFrom=fulltext)
* [2016 Frequency and Complexity of De Novo Structural Mutation in Autism](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4833290/)
* [2016 Evaluation of somatic copy number estimation tools for whole-exome sequencing data ](https://academic.oup.com/bib/article-abstract/17/2/185/1744035/Evaluation-of-somatic-copy-number-estimation-tools?redirectedFrom=fulltext)
* [2016 Detection of Clinically Relevant Copy Number Variants with Whole-Exome Sequencing](http://www.nature.com/gim/journal/vaop/ncurrent/full/gim2016163a.html)
* [2016 Sacral agenesis: a pilot whole exome sequencing and copy number study](http://bmcmedgenet.biomedcentral.com/articles/10.1186/s12881-016-0359-2)
* [2016 Assessing the reproducibility of exome copy number variations predictions](http://genomemedicine.biomedcentral.com/articles/10.1186/s13073-016-0336-6)
* [2016 Statistical models for DNA copy number variation detection using read-depth data from next generation sequencing experiments](http://onlinelibrary.wiley.com/doi/10.1111/anzs.12175/abstract)
* [2016 Patterns of genic intolerance of rare copy number variation in 59,898 human exomes](http://www.nature.com/ng/journal/v48/n10/abs/ng.3638.html)
* [2016 Challenges in detecting genomic copy number aberrations using next-generation sequencing data and the eXome Hidden Markov Model: a clinical exome-first diagnostic approach](https://www.nature.com/articles/hgv201625)
* [2016 Statistical models for DNA copy number variation detection using read-depth data from next generation sequencing experiments](http://onlinelibrary.wiley.com/doi/10.1111/anzs.12175/abstract?systemMessage=Wiley+Online+Library+usage+report+download+page+will+be+unavailable+on+Friday+24th+November+2017+at+21%3A00+EST+%2F+02.00+GMT+%2F+10%3A00+SGT+%28Saturday+25th+Nov+for+SGT+)
* [2016 Homozygous and hemizygous CNV detection from exome sequencing data in a Mendelian disease cohort](https://academic.oup.com/nar/article/45/4/1633/2681735)
* [2017 Profiling copy number variation and disease associations from 50,726 DiscovEHR Study exomes](http://biorxiv.org/content/early/2017/03/22/119461)
* [2017 Validation of copy number variation analysis for next-generation sequencing diagnostics](http://dx.doi.org/10.1038/ejhg.2017.42)
* [2017 Evaluation of three read-depth based CNV detection tools using whole-exome sequencing data](https://molecularcytogenetics.biomedcentral.com/articles/10.1186/s13039-017-0333-5)
* [2017 Clinical relevance of small copy-number variants in chromosomal microarray clinical testing](https://www.nature.com/articles/gim2016132)
* [2017 Germline copy number variations are associated with breast cancer risk and prognosis](https://www.nature.com/articles/s41598-017-14799-7)
* [2017 Evaluation of copy-number variants as modifiers of breast and ovarian cancer risk for BRCA1 pathogenic variant carriers](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5386423/)
* [2017 Copy Number Variations Detection: Unravelling the Problem in Tangible Aspects](https://www.ncbi.nlm.nih.gov/pubmed/27295681)
* [2017 Germline copy number variations are associated with breast cancer risk and prognosis](https://www.nature.com/articles/s41598-017-14799-7?utm_source=Springer_Nature_website&utm_medium=Referral_link&utm_content=ChrBigs-SN-OD-Genetics-Global&utm_campaign=SNOR_USG_ASHG18_LP)
* [2017 Detection of Copy Number Variations (CNVs) Based on the Coverage Depth from the Next Generation Sequencing Data](https://link.springer.com/chapter/10.1007%2F978-3-319-56418-0_2)
* [2017 Novel copy number variation of POMGNT1 associated with muscle-eye-brain disease detected by next-generation sequencing](https://www.nature.com/articles/s41598-017-07349-8)
> An example of manual calculation of CNV ratio
> "The NGS data were analyzed for detection of CNV by comparing the number of sequence reads between patient and control samples. First, the number of each exon sequence reads from patient and control was counted. For the purposes of this comparison, we used sequencing data from 20 control samples, including both males and females, without POMGNT1 deletions (genomic sequencing was performed in these patients for diagnosis of other diseases). Data were obtained from samples using the same sequencing chip, sequenced on the same Illumina platform, and trimmed to the same read length as each patient sample. Next, the patient/control ratios were calculated by dividing the number of each exon sequence reads from patient by the average number of each exon sequence reads from the 20 control samples. Patient/control ratio lower than 0.65 indicates heterozygous deletions and ratios higher than 1.3 indicates duplications. Statistical analyses and graphs were obtained using GraphPad Prism7 (GraphPad Software, La Jolla, CA, USA)."
* [2018 A randomized approach to speed up the analysis of large-scale read-count data in the application of CNV detection](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2077-6)
* [2018 Whole-exome sequence analysis highlights the role of unmasked recessive mutations in copy number variants with incomplete penetrance](https://www.nature.com/articles/s41431-018-0124-4)
* [2018 Copy Number Variants - Methods and Protocols](https://link.springer.com/book/10.1007/978-1-4939-8666-8)
* [2018 Assessment of the incorporation of CNV surveillance into gene panel next-generation sequencing testing for inherited retinal diseases](https://jmg.bmj.com/content/55/2/114)
* [2018 Rare Copy Number Variations and Predictors in Children With Intellectual Disability and Epilepsy](https://www.frontiersin.org/articles/10.3389/fneur.2018.00947/full)
* [2018 Whole-genome sequencing analysis of CNV using low-coverage and paired-end strategies is efficient and outperforms array-based CNV analysis](https://jmg.bmj.com/content/55/11/735)


## Long read
* [2017 Mapping and phasing of structural variation in patient genomes using nanopore sequencing](https://www.nature.com/articles/s41467-017-01343-4) - [github](https://github.com/mroosmalen/nanosv)
* [2018 NextSV: a meta-caller for structural variants from low-coverage long-read sequencing data](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2207-1)

## WGS
* [FindSV](https://github.com/J35P312/FindSV): FindSV is a structural variation pipeline written in nextflow and python. FindSV performs variant calling using TIDDIT and CNVnator, and Manta.
* [SvABA](https://github.com/walaj/svaba): Structural variation and indel analysis by assembly.
* [2007 PennCNV: An integrated hidden Markov model designed for high-resolution copy number variation detection in whole-genome SNP genotyping data](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2045149/)
* [2009 Sensitive and accurate detection of copy number variants using read depth of coverage](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2752127/)
* [2009 Personalized Copy-Number and Segmental Duplication Maps using Next-Generation Sequencing](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2875196/)
* [2010 Detecting copy number variation with mated short reads](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2963824/) - [CNVer github](https://github.com/pashadag/Cnver)
* [2011 CNVnator: An approach to discover, genotype, and characterize typical and atypical CNVs from family and population genome sequencing](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3106330/) - [CNVnator github](https://github.com/abyzovlab/CNVnator)
* [2011 Detecting common copy number variants in high-throughput sequencing data by using JointSLM algorithm](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3105418/)
* [2011 ReadDepth: A Parallel R Package for Detecting Copy Number Alterations from Short Sequencing Reads](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0016327)
* [2012 DELLY: structural variant discovery by integrated paired-end and split-read analysis](https://academic.oup.com/bioinformatics/article/28/18/i333/245403/DELLY-structural-variant-discovery-by-integrated) - [github](https://github.com/dellytools/delly): Delly2 was the best sv caller in the [DREAM challenge]()
* [2012 cn.MOPS: mixture of Poissons for discovering copy number variations in next-generation sequencing data with a low false discovery rate](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3351174/)
* [2012 Control-FREEC: a tool for assessing copy number and allelic content using next-generation sequencing data](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3268243/) - [Control-FREEC github](https://github.com/BoevaLab/FREEC)
* [2013 CNVeM: Copy Number Variation Detection Using Uncertainty of Read Mapping](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3590897/)
* [2014 LUMPY: a probabilistic framework for structural variant discovery](http://genomebiology.biomedcentral.com/articles/10.1186/gb-2014-15-6-r84) - [github](https://github.com/arq5x/lumpy-sv)
* [2014 The CNVrd2 package: measurement of copy number at complex loci using high-throughput sequencing data](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4117933/)
* [2015 Wham: Identifying Structural Variants of Biological Consequence](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004572) - [github](https://github.com/zeeev/wham) - [mergeSVcallers](https://github.com/zeeev/mergeSVcallers)
* [2015 MetaSV: an accurate and integrative structural-variant caller for next generation sequencing](https://academic.oup.com/bioinformatics/article/31/16/2741/321286/MetaSV-an-accurate-and-integrative-structural) - [github](http://bioinform.github.io/metasv/)
* [2015 GROM-RD: resolving genomic biases to improve read depth detection of copy number variants](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4369336/)
* [2016 Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications](https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btv710) - [github](https://github.com/Illumina/manta)
* [2016 FACETS: allele-specific copy number and clonal heterogeneity analysis tool for high-throughput DNA sequencing](https://academic.oup.com/nar/article/44/16/e131/2460163)
> ... an ASCN tool and open-source software with a broad application to whole genome, whole-exome, as well as targeted panel sequencing platforms. 

* [2017 SV2: Accurate Structural Variation Genotyping and De Novo Mutation Detection](http://biorxiv.org/content/early/2017/03/17/113498) - [github](https://github.com/dantaki/SV2)
* [2017 GRIDSS: sensitive and specific genomic rearrangement detection using positional de Bruijn graph assembly](http://biorxiv.org/content/early/2017/02/21/110387) - [github](https://github.com/PapenfussLab/gridss)
* [2017 Detection of complex structural variation from paired-end sequencing data](https://www.biorxiv.org/content/early/2017/10/08/200170?rss=1)
* [2017 CLImAT-HET: detecting subclonal copy number alterations and loss of heterozygosity in heterogeneous tumor samples from whole-genome sequencing data](https://bmcmedgenomics.biomedcentral.com/articles/10.1186/s12920-017-0255-4)
* [2017 Performance of four modern whole genome amplification methods for copy number variant detection in single cells](https://www.nature.com/articles/s41598-017-03711-y?utm_source=Springer_Nature_website&utm_medium=Referral_link&utm_content=ChrBigs-SN-OD-Genetics-Global&utm_campaign=SNOR_USG_ASHG18_LP)
* [2018 Detection of complex structural variation from paired-end sequencing data](https://www.biorxiv.org/content/early/2018/02/01/200170?rss=1)
* [2018 GIGGLE: a search engine for large-scale integrated genome analysis](https://www.nature.com/articles/nmeth.4556)
* [2018 FusorSV: an algorithm for optimally combining data from multiple structural variation detection methods](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1404-6)
* [2018 SvABA: genome-wide detection of structural variants and indels by local assembly](https://genome.cshlp.org/content/28/4/581.abstract?platform=hootsuite)
* [2018 Global characterization of copy number variants in epilepsy patients from whole genome sequencing](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007285)
* [2018 Human copy number variants are enriched in regions of low mappability](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky538/5042037#.Wy09sMRev6o.twitter)
* [2018 MSeq-CNV: accurate detection of Copy Number Variation from Sequencing of Multiple samples](https://www.nature.com/articles/s41598-018-22323-8)
* [2018 CoNVaQ: a web tool for copy number variation-based association studies](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4732-8?utm_source=Springer_Nature_website&utm_medium=Referral_link&utm_content=ChrBigs-SN-OD-Genetics-Global&utm_campaign=SNOR_USG_ASHG18_LP)
* [2018 Using SAAS-CNV to Detect and Characterize Somatic Copy Number Alterations in Cancer Genomes from Next Generation Sequencing and SNP Array Data](https://link.springer.com/protocol/10.1007%2F978-1-4939-8666-8_2)
* [2018 Using HaMMLET for Bayesian Segmentation of WGS Read-Depth Data](https://link.springer.com/protocol/10.1007/978-1-4939-8666-8_6)
* [2018 Split-Read Indel and Structural Variant Calling Using PINDEL](https://link.springer.com/protocol/10.1007/978-1-4939-8666-8_7)
* [2018 Detecting Small Inversions Using SRinversion](https://link.springer.com/protocol/10.1007/978-1-4939-8666-8_8)
* [2018 Detection of CNVs in NGS Data Using VS-CNV](https://link.springer.com/protocol/10.1007/978-1-4939-8666-8_9)
* [2018 Structural Variant Breakpoint Detection with novoBreak](https://link.springer.com/protocol/10.1007/978-1-4939-8666-8_10)
* [2018 Versatile Identification of Copy Number Variants with Canvas](https://link.springer.com/protocol/10.1007/978-1-4939-8666-8_12)
* [2018 Identification of Copy Number Variants from SNP Arrays Using PennCNV](https://link.springer.com/protocol/10.1007/978-1-4939-8666-8_1)
* [2018 Whole-Genome Shotgun Sequence CNV Detection Using Read Depth](https://link.springer.com/protocol/10.1007/978-1-4939-8666-8_4)
* [2018 iCopyDAV: Integrated platform for copy number variations—Detection, annotation and visualization](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0195334) - [iCopyDAV github](https://github.com/vogetihrsh/icopydav)

## WES
* [2012 Copy number variation detection and genotyping from exome sequence data](https://genome.cshlp.org/content/22/8/1525)
* [2015 CODEX: a normalization and copy number variation detection method for whole exome sequencing](https://academic.oup.com/nar/article/43/6/e39/2453417/CODEX-a-normalization-and-copy-number-variation)
* [2015 Detecting copy-number variations in whole-exome sequencing data using the eXome Hidden Markov Model: an ‘exome-first’ approach](https://www.nature.com/articles/jhg2014124)
* [2016 Enhanced copy number variants detection from whole-exome sequencing data using EXCAVATOR2](https://academic.oup.com/nar/article/44/20/e154/2607979/Enhanced-copy-number-variants-detection-from-whole)
* [2016 CLAMMS: a scalable algorithm for calling common and rare copy number variants from exome sequencing data](https://academic.oup.com/bioinformatics/article/32/1/133/1743911/CLAMMS-a-scalable-algorithm-for-calling-common-and) - [github](https://github.com/rgcgithub/clamms)
> [KaryoScan: abnormal karyotype detection from whole-exome sequence](https://www.biorxiv.org/content/early/2017/10/17/204719) - [github](https://github.com/rgcgithub/karyoscan)
* [2016 Challenges in detecting genomic copy number aberrations using next-generation sequencing data and the eXome Hidden Markov Model: a clinical exome-first diagnostic approach](https://www.nature.com/articles/hgv201625)
* [2016 CloneCNA: detecting subclonal somatic copy number alterations in heterogeneous tumor samples from whole-exome sequencing data](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1174-7)
* [2016 CNV-RF Is a Random Forest–Based Copy Number Variation Detection Method Using Next-Generation Sequencing](https://jmd.amjpathol.org/article/S1525-1578(16)30131-3/fulltext)
* [2017 ExCNVSS: A Noise-Robust Method for Copy Number Variation Detection in Whole Exome Sequencing Data](https://www.hindawi.com/journals/bmri/2017/9631282/)
* [2017 Anaconda: AN automated pipeline for somatic COpy Number variation Detectio and Annotation from tumor exome sequencing data](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1833-3)
* [2017 An evaluation of copy number variation detection tools for cancer using whole exome sequencing data](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1705-x)
* [2017 WISExome: a within-sample comparison approach to detect copy number variations in whole exome sequencing data](https://www.nature.com/articles/s41431-017-0005-2)
* [2018 A Comprehensive Workflow for Read Depth-Based Identification of Copy-Number Variation from Whole-Genome Sequence Data](http://www.cell.com/ajhg/abstract/S0002-9297(17)30496-2) - [github](https://github.com/bjtrost/TCAG-WGS-CNV-workflow)
* [2018 LinkedSV: Detection of mosaic structural variants from linked-read exome and genome sequencing data](https://www.biorxiv.org/content/early/2018/10/10/409789)
* [2019 Systematic and comprehensive benchmarking of an exome sequencing based germline copy-number variant detection pipeline in a clinical setting](https://www.biorxiv.org/content/10.1101/579755v1.abstract)
* [2019 Comprehensively benchmarking applications for detecting copy number variation](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007069)

## AS 
* [2012 A robust model for read count data in exome sequencing experiments and implications for copy number variant calling](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3476336/)
* [2013 CNV-TV: A robust method to discover copy number variation from short sequencing reads](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-150)
* [2015 VisCap: inference and visualization of germ-line copy-number variants from targeted clinical sequencing data](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4940431/)
* [2015 CopywriteR: DNA copy number detection from off-target sequence data](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0617-1)
* [2016 CoNVaDING: Single Exon Variation Detection in Targeted NGS Data](http://onlinelibrary.wiley.com/doi/10.1002/humu.22969/abstract?systemMessage=Wiley+Online+Library+will+be+unavailable+on+Saturday+7th+Oct+from+03.00+EDT+%2F+08%3A00+BST+%2F+12%3A30+IST+%2F+15.00+SGT+to+08.00+EDT+%2F+13.00+BST+%2F+17%3A30+IST+%2F+20.00+SGT+and+Sunday+8th+Oct+from+03.00+EDT+%2F+08%3A00+BST+%2F+12%3A30+IST+%2F+15.00+SGT+to+06.00+EDT+%2F+11.00+BST+%2F+15%3A30+IST+%2F+18.00+SGT+for+essential+maintenance.+Apologies+for+the+inconvenience+caused+.)
* [2016 Accurate clinical detection of exon copy number variants in a targeted NGS panel using DECoN](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5409526/)
* [2016 PureCN: copy number calling and SNV classification using targeted short read sequencing](https://scfbm.biomedcentral.com/articles/10.1186/s13029-016-0060-z) - [github](https://github.com/lima1/PureCN)
* [2017 SeqCNV: a novel method for identification of copy number variations in targeted next-generation sequencing data](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1566-3)
* [2017 panelcn.MOPS: Copy-number detection in targeted NGS panel data for clinical diagnostics](http://dx.doi.org/10.1002/humu.23237)
* [2017 An enhanced method for targeted next generation sequencing copy number variant detection using ExomeDepth](https://wellcomeopenresearch.org/articles/2-49/v1)
* [2017 CODEX2: full-spectrum copy number variation detection by high-throughput DNA sequencing](https://www.biorxiv.org/content/early/2017/10/30/211698)
* [2017 Clinical Validation of Copy Number Variant Detection from Targeted Next-Generation Sequencing Panels](http://jmd.amjpathol.org/article/S1525-1578(17)30207-6/fulltext)
* [2017 ‘COV’COP’ allows to detect CNVs responsible for inherited diseases among amplicons sequencing data](https://academic.oup.com/bioinformatics/article/33/10/1586/2962858) - [git](https://git.unilim.fr/merilp02/CovCop2)
* [2017 CNVkit: Genome-Wide Copy Number Detection and Visualization from Targeted DNA Sequencing](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004873) - [website](https://cnvkit.readthedocs.io/en/stable/)
* [2018 Ioncopy: an R Shiny app to call copy number alterations in targeted NGS data](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2159-5)
* [2018 Validation of an ultra-fast CNV calling tool for Next Generation Sequencing data using MLPA-verified copy number alterations](https://www.biorxiv.org/content/early/2018/06/06/340505)
* [2018 DeviCNV: detection and visualization of exon-level copy number variants in targeted next-generation sequencing data](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2409-6)
* [2018 Ximmer: a system for improving accuracy and consistency of CNV calling from exome data](https://academic.oup.com/gigascience/article/7/10/giy112/5091801)

## RNAseq
* [2018 CaSpER: Identification, visualization and integrative analysis of CNV events in multiscale resolution using single-cell or bulk RNA sequencing data](https://www.biorxiv.org/content/early/2018/09/26/426122)
* [2018 CNVkit-RNA: Copy number inference from RNA-Sequencing data](https://www.biorxiv.org/content/early/2018/09/04/408534?rss=1) - [youtube tutorial](https://www.youtube.com/watch?v=JGOIXYoLG6w&feature=youtu.be)

## Annotation
* [StructuralVariantAnnotation](https://github.com/PapenfussLab/StructuralVariantAnnotation): contains useful helper functions for dealing with structural variants in VCF format.
* [2015 SpeedSeq: ultra-fast personal genome analysis and interpretation](http://www.nature.com/nmeth/journal/v12/n10/full/nmeth.3505.html) - SVTyper - [github](https://github.com/hall-lab/svtyper)
* [2016 SVScore: an impact prediction tool for structural variation](https://academic.oup.com/bioinformatics/article/doi/10.1093/bioinformatics/btw789/2748212/SVScore-an-impact-prediction-tool-for-structural) - [github](https://github.com/lganel/SVScore)
* [2016 Prioritisation of Structural Variant Calls in Cancer Genomes](http://biorxiv.org/content/early/2016/11/04/084640) - [github](https://github.com/AstraZeneca-NGS/simple_sv_annotation)
* [2016 cnvScan: a CNV screening and annotation tool to improve the clinical utility of computational CNV prediction from exome sequencing data](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-016-2374-2)
* [2017 Annotation Education Series: CNV Annotations](http://blog.goldenhelix.com/dkammeraad/annotation-education-series-final-chapter-cnv-annotations/)
* [2018 AnnotSV: an integrated tool for structural variations annotation](https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/bty304/4970516?redirectedFrom=fulltext) - [website](http://www.lbgi.fr/AnnotSV/)
* [2018 VarAFT: a variant annotation and filtration system for human next generation sequencing data](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6030844/)
* [2019 An open resource of structural variation for medical and population genetics](https://www.biorxiv.org/content/10.1101/578674v1)
* [2019 Duphold: scalable, depth-based annotation and curation of high-confidence structural variant calls](https://academic.oup.com/gigascience/article/8/4/giz040/5477467)

## Visualization
* [CNVplot](https://github.com/dantaki/CNVplot): Plot CNV data with a genome viewer in R.
* [cnvgram](https://github.com/cc2qe/cnvgram): Draw CNV diagrams.
* [Stupid Simple Structural Variant View](https://github.com/ryanlayer/svv): A two-step process that can help visualize the coverage near a variant across multiple BAMs.
* [CNView](https://github.com/RCollins13/CNView): Visualization, quantitation, and annotation of CNVs from population-scale whole-genome sequencing data.
* [2015 svviz: a read viewer for validating structural variants](https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btv478) - [github](https://github.com/svviz/svviz)
* [2016 Ribbon: Visualizing complex genome alignments and structural variation](http://biorxiv.org/content/early/2016/10/20/082123) - [github](https://github.com/MariaNattestad/Ribbon)
* [samplot:](https://github.com/ryanlayer/samplot)

## Orchestrators
* [CNV-patissier github](https://github.com/stefpiatek/cnv-patissier)
* [Ximmer github](https://github.com/ssadedin/ximmer)
* [megSAP github](https://github.com/imgag/megSAP)

## Others
* [SVsim](https://github.com/GregoryFaust/SVsim): a tool that generates synthetic Structural Variant calls as benchmarks to test/evaluate SV calling pipelines.
* [Structural Variant Catalog](https://github.com/tobiasrausch/svcatalog): A repository for human genetic structural variants (SVs) discovered by Delly in the 1000 Genomes cohort of samples.
* [SVDB](https://github.com/J35P312/SVDB): SVDB is a toolkit for constructing and querying structural variant databases. The databases are constructed using the output vcf files from structural variant callers such as TIDDIT, Manta, Fermikit or Delly. [The thousand genomes structural variant calls may also be used as a database](ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase3/integrated_sv_map/)
* [bcftools cnv](https://samtools.github.io/bcftools/bcftools.html#cnv)
* [2017 Bamgineer: Introduction of simulated allele-specific copy number variants into exome and targeted sequence data sets](https://www.biorxiv.org/content/early/2017/03/22/119636) - [github](https://github.com/pughlab/bamgineer)
* [2017 CNVbase: Batch identification of novel and rare copy number variations based on multi-ethnic population data](https://www.sciencedirect.com/science/article/pii/S167385271730108X?via%3Dihub) - [CNVbase](http://database.gdg-fudan.org/DB_HTML/Home.html)
* [2018 SVEngine: an efficient and versatile simulator of genome structural variations with features of cancer clonal evolution](https://www.biorxiv.org/content/early/2018/01/12/247536)
* [2018 RBV: Read balance validator, a tool for prioritising copy number variations in germline conditions](https://www.biorxiv.org/content/early/2018/06/07/340166) - [github](https://github.com/whitneywhitford/RBV)
* [2018 Detection of de novo copy number deletions from targeted sequencing of trios](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/bty677/5064107)
* [2018 Bamgineer: Introduction of simulated allele-specific copy number variants into exome and targeted sequence data sets](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006080) - [github](https://github.com/pughlab/bamgineer)
* [2018 Copy-number variants in clinical genome sequencing: deployment and interpretation for rare and undiagnosed disease](https://www.nature.com/articles/s41436-018-0295-y)
